### PR TITLE
Additions and clarifications to the origin trial article.

### PR DESCRIPTION
**Pull request by jpmedley**
@samdutton I've made a few suggestions based on my own experience with and knowledge of origin trials.

### CODE REVIEW

File: src/site/content/en/blog/origin-trials/index.md [hunk 1]
```diff
@@ -17,17 +17,21 @@ Origin trials enable you to build functionality your users can try for the
...
+they may need to upgrade). Likewise, your developers can try out demos and
+prototypes. Origin trials also help Chrome engineers understand how new features
+are used, and how they may interact with other web technologies.
 
 Origin trials are public and open to all developers, and limited in duration and
 usage. Additionally, origin trials are a self-managed process with limited
 documentation and support. Participants should be willing and able to work
-relatively independently using the documentation available.
+relatively independently using the documentation available, which, at this stage,
+will likely be limited to specs and explainers. Though, web.dev tries to provide
```
Comment (samdutton) [inline src/site/content/en/blog/origin-trials/index.md:29]: I don't think beginning the sentence with 'Though'. 

Maybe

Comment (jpmedley) [inline src/site/content/en/blog/origin-trials/index.md:29]:
```suggestion
will likely be limited to specs and explainers, though, web.dev tries to provide
```

Comment (jpmedley) [reply src/site/content/en/blog/origin-trials/index.md]: Done.

File: src/site/content/en/blog/origin-trials/index.md [hunk 2]
```diff
@@ -17,17 +17,21 @@ Origin trials enable you to build functionality your users can try for the
...
+they may need to upgrade). Likewise, your developers can try out demos and
+prototypes. Origin trials also help Chrome engineers understand how new features
+are used, and how they may interact with other web technologies.
 
 Origin trials are public and open to all developers, and limited in duration and
 usage. Additionally, origin trials are a self-managed process with limited
 documentation and support. Participants should be willing and able to work
-relatively independently using the documentation available.
+relatively independently using the documentation available, which, at this stage,
+will likely be limited to specs and explainers, though, web.dev tries to provide
```
Comment (jpmedley) [inline src/site/content/en/blog/origin-trials/index.md:29]:
```suggestion
will likely be limited to specs and explainers, though web.dev tries to provide
```

**Pull request merged**